frontend/ui/data/keyboardlayouts/std.lua

```diff
@@ -8,7 +17,7 @@ return {
         {  --  1           2       3       4       5       6       7       8       9       10      11      12
             { "Q",        "q",    "„",    "0",    "Й",    "й",    "?",    "!",    "Å",    "å",    "1",    "ª", },
             { "W",        "w",    "!",    "1",    "Ц",    "ц",    "(",    "1",    "Ä",    "ä",    "2",    "º", },
-            { "E",        "e",    "@",    "2",    "У",    "у",    ")",    "2",    "Ö",    "ö",    "3",    "¡", },
+            { "E",          e,    "@",    "2",    "У",    "у",    ")",    "2",    "Ö",    "ö",    "3",    "¡", },
```

Comment:
or name this local variable (and the future ones) `_e_`, so it's balanced in there, and no risk of confusion (and luacheck complain?) when we use `for k, v in pairs() do` and we implement swipes for the letters k or v ?

Reply:
I'm not entirely sure what you mean. It's drop-in and all that's needed is keyboard updates. ;-)

(Did you say k? I think `ķ` is about the only one possible. But of course such keys would also be good for punctuation characters.)

Reply:
I meant just what you just changed, `local _e_ =` on the first line, so nothing else :)
(this `k` thing was just an example of another common single char local variable; `i` in `for i, v in pairs()` would have been another better example, as we could have `ï` `î` :)